site/content/docs/5.1/examples/card/card.css

```diff
@@ -0,0 +1,4 @@
+.card-main{
```

Comment:
see Card section [Background and color](https://getbootstrap.com/docs/5.1/components/card/#background-and-color)

Reply:
@WinterSilence, Thanks again for the suggestion. The main reason was to use custom class and CSS for it. 
BTW, Now I have updated these sections with bootstrap background and color. So now both are exist (custom class + Bootstrap class)